test/reg-test/fam-api-reg/fam_context_mt_reg_test.cpp

```diff
@@ -49,12 +49,12 @@ Fam_Options fam_opts;
 Fam_Region_Descriptor *testRegionDesc;
 const char *testRegionStr;
 int rc;
-#define NUM_THREADS 10
+#define NUM_THREADS 7
 #define REGION_SIZE (32 * 1024 * NUM_THREADS)
 #define REGION_PERM 0777
 #define NUM_ITERATIONS 100
 #define NUM_IO_ITERATIONS 5
-#define NUM_CONTEXTS 256
+#define NUM_CONTEXTS 7

```

Comment:
Please add a comment, saying to increase number of contexts, we need to increase number of open files.

Reply:
Sure, will do that